url_test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
```

Comment:
Yes, url_testing is working fine. Checked on the build url. Gives 404 error, Catched it and correctly showing the ISO Building message